Lib/test/test_xmlrpc.py

```diff
@@ -343,6 +343,78 @@ def run_server():
             self.assertEqual(p.method(), 5)
             self.assertEqual(p.method(), 5)
 
+    def test_simple_xml_rpc_dispatcher_resolves_registered_func_and_does_not_chain_excs_when_calling_it(self):
+        exp_params = 1, 2, 3
+        dispatched = False
+
+        class DispatchExc(Exception):
```

Comment:
You will need to specify long qualified name (`SimpleXMLRPCDispatcherTestCase.DispatcherExc`) or use outer `self` to access it.